### PR TITLE
feat: add kinematics data

### LINKED ISSUE #4: kinematics data from Liew et al

### PR DESCRIPTION
**Pull request by m-muecke**
Closes: https://github.com/tidyfun/tidyfundata/issues/4

### CODE REVIEW

**Comment by m-muecke** [comment]
@fabian-s should we reference one of the papers for the data such as https://arxiv.org/pdf/2312.05523?
And do you want the categorical variables to be converted as factors like they're right now or keep them as characters?
Lastly, are you fine with the name `kinematics` for the dataset?

**Comment by fabian-s** [comment]
@davidruegamer

if we use these data in the package, probably best to cite 

image

right? 
which one?

**Comment by fabian-s** [comment]
> want the categorical variables to be converted as factors like they're right now or keep them as characters?

def. factor, thanks!

>  Lastly, are you fine with the name `kinematics` for the dataset?

not sure -- maybe `running` or `loaded_run` would be more descriptive ?

**Comment by m-muecke** [comment]
> > want the categorical variables to be converted as factors like they're right now or keep them as characters?
> 
> def. factor, thanks!
> 
> > Lastly, are you fine with the name `kinematics` for the dataset?
> 
> not sure -- maybe `running` or `loaded_run` would be more descriptive ?

perfect, whatever name you prefer

**Comment by davidruegamer** [comment]
@fabian-s There is an `id` in the dataset indicating from which study each observation is. If you use both (`bernard_study1_...` and `bernd_study2_...`) I guess citing both makes sense. In addition, if you plan to use data with `id` starting with `fukuchi_`, then I guess you would also have to cite 

    @article{fukuchi2017public,
        title={A public dataset of running biomechanics and the effects of running speed on lower extremity kinematics and kinetics},
        author={Fukuchi, Reginaldo K and Fukuchi, Claudiane A and Duarte, Marcos},
        journal={PeerJ},
        volume={5},
        pages={e3298},
        year={2017},
        publisher={PeerJ Inc.}
    }

**Comment by m-muecke** [comment]
@fabian-s should we then add the data from all studies and add Fukuchi to the citation or only add some of the studies?

**Comment by fabian-s** [comment]
yeah, that makes sense I think!

**Comment by m-muecke** [comment]
> yeah, that makes sense I think! 

So, whole dataset?

**Comment by fabian-s** [comment]
sorry, yes: whole dataset?

**Comment by fabian-s** [comment]
thx @m-muecke  -- 

@davidruegamer sorry 2 more questions:  

- what exactly is the "step number" covariate in this dataset? did you use this for any analyses?  
- I think we'll discard the "validation set" covariate (train/test), don't see how that would be relevant for a broader audience -- agree?

**Comment by davidruegamer** [comment]
@fabian-s 
- I think `step` is related to the number of steps as these can vary depending on the height of the participant, speed of the treadmill, etc. I remember including speed, weight, and height in some of our analyses, where the step number was not available as a feature. So for realistic modeling applications, I guess including `step` could make sense.
- agree
